.gitignore

```diff
@@ -6,3 +6,4 @@ dcrpool
 logs
 *.log
 *.conf
+/pool/testdb
```

Comment:
testdb is deleted after tests, not sure why it should be added here.

Reply:
The testdb is indeed removed successfully, however I wanted to add this to .gitignore because I often run tests whilst reviewing my own changes, so the testdb shows up in my git diffs.